src/main/java/frc/robot/swerve/SwerveWheel.java

```diff
@@ -46,11 +46,7 @@ public SwerveWheel(String name, int drivePort, int turnPort, boolean inverted) {
         this.currentState = new SwerveModuleState();
         this.desiredState = currentState;
 
-        this.driveMotor = new TalonFX(drivePort);
-        this.driveMotor.setNeutralMode(SwerveConfig.DEFAULT_DRIVE_NEUTRAL_MODE);
-        this.driveMotor.setInverted(inverted ? InvertType.InvertMotorOutput : InvertType.None);
-        this.driveMotor.configClosedloopRamp(SwerveConfig.DEFAULT_DRIVE_RAMP_RATE);
-        SwerveConfig.DRIVE_PID.configPID(driveMotor);
+        this.driveMotor = SwerveConfig.createDriveMotor(drivePort, inverted);
```

Comment:
This is how we'll actually use your method to clean this up. Notice how many lines of code we can delete!